HousingManagementSystemApi/Properties/launchSettings.json

```diff
@@ -28,6 +28,8 @@
         "AUTHENTICATION_IDENTIFIER": "<REPLACE WITH A UNIQUE AUTHENTICATION IDENTIFIER>",
         "JWT_SECRET" : "<REPLACE WITH A SECRET>",
         "SENTRY_DNS": "https://<REPLACE WITH SENTRY DSN>",
+        "HOUSING_ASSET_API_URI": "https://<REPLACE WITH API URI>",
```

Comment:
Serverless.yml will need to have these environment variables assigned via AWS Parameter Store too.